Makefile

```diff
@@ -152,15 +152,22 @@ codegen:
 DOCKER_CMD := $(CONTAINER_RUNTIME) run \
 	-it \
 	--rm \
-	-v gomodcache:/go/pkg/mod \
+	-v gomodcache:/home/user/gocache \
 	-v $(dir $(realpath $(firstword $(MAKEFILE_LIST)))):/workspaces/kargo \
 	-v /workspaces/kargo/ui/node_modules \
 	-w /workspaces/kargo \
 	kargo:dev-tools
 
+DEV_TOOLS_BUILD_OPTS =
+ifeq ($(GOOS),linux)
```

Comment:
`GOOS` is often undefined. Would it make sense to use `uname` here instead?

Reply:
`$(GOOS)` != `${GOOS}`, and actually is based on `uname` already. See: https://github.com/akuity/kargo/blob/2bb3b3895bfb2d0d3bf77242a5e584b05aed1c8f/Makefile#L43 😄

Reply:
Apologies... I overlooked that.